packages/next-codemod/bin/upgrade.ts

```diff
@@ -228,7 +228,7 @@ async function suggestCodemods(
     {
       type: 'multiselect',
       name: 'codemods',
-      message: `\nThe following ${chalk.blue('codemods')} are recommended for your upgrade. Would you like to apply them?`,
+      message: `The following ${chalk.blue('codemods')} are recommended for your upgrade. Would you like to apply them?`,
```

Comment:
The current prompt makes it seem like it'll automatically apply them all but I don't think it will (but ignore me if I'm wrong)

```suggestion
      message: `The following ${chalk.blue('codemods')} are recommended for your upgrade. Select which ones to apply.`,
```